dotcom-rendering/src/palette.ts

```diff
@@ -5630,6 +5779,10 @@ const paletteColours = {
 		light: articleBackgroundLight,
 		dark: articleBackgroundDark,
 	},
+	'--article-inner-background': {
```

Comment:
are we actually using this anywhere? Would we maybe want to use the regular article background and add explicit colours for audio/video?

Reply:
This is to achieve the very slight difference in background color with the main background
<img width="441" alt="Screenshot 2024-04-30 at 13 13 30" src="https://github.com/guardian/dotcom-rendering/assets/1731150/b0dd94df-8877-4efe-ba7a-84bbf530bb5a">